raiden/tasks.py

```diff
@@ -476,16 +473,84 @@ def _run(self):
                 pex(transfer.target),
             )
 
+        maximum_expiration = (
+            originating_channel.settle_timeout +
```

Comment:
This is also my preferred style, changed because of the flake8 tool.

The PEP8 Style Guide was updated on 14/Apr https://mail.python.org/pipermail/python-ideas/2016-April/thread.html#39752